types/react-native-video/index.d.ts

```diff
@@ -141,6 +141,7 @@ export interface VideoProperties extends ViewProps {
     automaticallyWaitsToMinimizeStalling?: boolean;
     paused?: boolean;
     muted?: boolean;
+    mixWithOthers?: boolean;
```

Comment:
Unless, I'm reading [the docs](https://github.com/react-native-video/react-native-video/blob/1eb11ce7b7619a04e65affedcb1ffff4f82f0af3/README.md#mixWithOthers) wrong, isn't this a string value, not a boolean?